routes/summary.go

```diff
@@ -41,6 +43,19 @@ func (h *SummaryHandler) RegisterRoutes(router chi.Router) {
 	router.Mount("/summary", r)
 }
 
+func (h *SummaryHandler) FetchSummaryForDailyProjectStats(params *models.SummaryParams) ([]*models.Summary, error) {
```

Comment:
Minor one: make this a private method (lower-case) and move it _below_ all public methods. Also, please stick to Go code style convention of having camel-case variables (e.g. `cur_summary` -> `curSummary`).